src/explore-education-statistics-frontend/src/modules/methodologies/__tests__/MethodologyPage.test.tsx

```diff
@@ -197,4 +197,56 @@ describe('MethodologyPage', () => {
       );
     });
   });
+
+  describe('Help and Support Section', () => {
+    const selectRelatedInformationSection = (): HTMLElement => {
+      const relatedInformation = screen.getByRole('heading', {
+        name: 'Related information',
+      }).parentElement;
+
+      if (!relatedInformation) {
+        throw new Error(
+          'Failing test early - the "Related information" section could not be found.',
+        );
+      }
+
+      return relatedInformation;
+    };
+
+    test('that it renders the Help and Support section', () => {
+      render(<MethodologyPage data={testMethodology} />);
+
+      expect(
+        within(selectRelatedInformationSection()).getByRole('heading', {
+          name: 'Help and support',
+        }),
+      ).toBeVisible();
```

Comment:
Same as other comments - would suggest using `toBeInTheDocument` instead.

Same below

Reply:
I'd rather keep `toBeVisible`. 

One of the acceptance criteria in the ticket explicitly states that the Contact Us section must not be hidden within an accordion. If we weren't using the React framework, accordions could be implemented by applying the `display: none` or `visibility: hidden` attributes, so I maintain it's a valid regression.

I also don't buy the argument that we should make our tests less robust for the sake of test run speed - I'd rather be safe.